### PR TITLE
Mm/recipe list display

### PR DESCRIPTION
**Pull request by mateuszmarc**
Move Login servlet class into `web` directory so it is are together with other servlets.
Create AdminRecipeList servlet.
Create admin-recipe-list.jsp:

I refactored servlets because of derived common app view part: header.jsp - Header needs to have access to `adminName` session attribute right away after admin is authenticated - otherwise we have to retrieve `adminName` in every servlet that uses view that includes `header.jsp.` I set session attribute `adminName` in Login servlet after admin authenticated. This means that we already have `adminName` and `adminId` in session so we don't have to use `AdminDao.read` method to retrieve `adminName` in every servlet .
Therefore I made following changes:

- Change Login servlet:
   - Set `adminName` session attribute along with `adminId` attribute if the admin is authenticated.
- Change `AddNewRecipe` servlet:
   - Remove private static final `AdminDao` field.
   - Remove `getSession` method in `doGet` method
   - Remove `getAttribute("adminId")` statement in doGet method
    - Remove setting session attribute for `adminName` from `adminDao.read` method in `doGet` method.
- Change Dashboard servlet:
    - Remove private static final `AdminDao` field.
    - Remove setting `adminName` session attribute by calling `adminDao.read` method.

### CODE REVIEW

File: src/main/java/pl/coderslab/web/Login.java [hunk 1]
```diff
@@ -1,15 +1,17 @@
-package pl.coderslab.model;
+package pl.coderslab.web;
+
+import lombok.extern.java.Log;
```
Comment (AnielskieOczko) [inline src/main/java/pl/coderslab/web/Login.java]: powinniśmy się zdecydować, którego loggera chcemy używać.

**Review by AnielskieOczko — COMMENTED** [review]
wygląda spoko, przetestowałem u siebie - działa bez zarzutu

File: src/main/java/pl/coderslab/web/Login.java (hunk 1 above)
Comment (mateuszmarc) [reply src/main/java/pl/coderslab/web/Login.java]: fakt, dotychczas używaliśmy loggera z dependencji  log4j2 - to usunę tego loggera z tego pliku.

File: src/main/java/pl/coderslab/web/HomeServlet.java [hunk 2]
```diff
@@ -1,17 +1,13 @@
 package pl.coderslab.web;
 
 import lombok.extern.java.Log;
```
Comment (AnielskieOczko) [inline src/main/java/pl/coderslab/web/HomeServlet.java]: tutaj jak możesz to też usuń

Comment (AnielskieOczko) [reply src/main/java/pl/coderslab/web/HomeServlet.java]: Po za tym nic nie znalazłem więc myślę, że jak nie ma więcej uwag to można mergować.

**Pull request merged**